VndbSharp/Structs/Models/VnList/VnList.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+
+namespace VndbSharp.Structs.Models.VnList
+{
+    [JsonObject]
+    public class VnList
+    {
+        [JsonProperty("vn")]
+        public Int32 VisualNovelId;
+        [JsonProperty("status")]// : TODO: Possibly change this to a byte, since only 4 values will be returned?
+        public Int32 Status;
+        [JsonProperty("added")]
+        public Int32 TimeAdded;
```

Comment:
This should probs be a `DateTime` object, using a `UnixTimestampConverter`. An example implementation that could be used, if you want: [UnixTimestampConverter.zip](https://github.com/Nikey646/VndbSharp/files/692048/UnixTimestampConverter.zip)